docs/en/features/census_configuration.md

```diff
@@ -1,155 +1,145 @@
+- **Path for Postal Code**: Path in the request structure that sends the Postal Code.
 
-    *NOTE: DO NOT FILL IN if the WebService does not require the Postal Code to verify a user.*
+  *NOTE: DO NOT FILL IN if the WebService does not require the Postal Code to verify a user.*
 
-    En el caso del *Example* lo dejaríamos en blanco, ya que no se necesita enviar el código postal para verificar a un usuario.
+  In *this example*, we will leave it blank, since it is not necessary to send the postal code to verify a user.
 
-    Example:
-    ![Request Data - Path postal code](../../img/remote_census/request-data-path-postal-code-en.png)
+  ![Request data - Path for postal code is left blank](../../img/remote_census/request-data-path-postal-code-en.png)
 
-1. **Response data**
+### Response data
 
-  In this section we will configure all the necessary fields to be able to receive the answer of the WebService and to verify a user in the application.
+In this section we will configure all the necessary fields to be able to receive the answer of the WebService and to verify a user in the application.
 
-  ![Response Data](../../img/remote_census/response-data-en.png)
+![Response data](../../img/remote_census/response-data-en.png)
 
-  As in the previous section we will define an example answer, to help you understand how to fill in each of the fields in this section.
+As in the previous section, we will define an example answer to help you understand how to fill in each of the fields in this section.
 
-  ```ruby
-      {
-        get_habita_datos_response: {
-          get_habita_datos_return: {
-            datos_habitante: {
-              item: {
-                fecha_nacimiento_string: "31-12-1980",
-                identificador_documento: "12345678Z",
-                descripcion_sexo: "Varón",
-                nombre: "José",
-                apellido1: "García"
-              }
-            },
-            datos_vivienda: {
-              item: {
-                codigo_postal: "28013",
-                codigo_distrito: "01"
-              }
-            }
-          }
+```ruby
+{
+  get_habita_datos_response: {
+    get_habita_datos_return: {
+      datos_habitante: {
+        item: {
+          fecha_nacimiento_string: "31-12-1980",
+          identificador_documento: "12345678Z",
+          descripcion_sexo: "Varón",
+          nombre: "José",
+          apellido1: "García"
+        }
+      },
+      datos_vivienda: {
+        item: {
+          codigo_postal: "28013",
+          codigo_distrito: "01"
         }
       }
-  ```
+    }
+  }
+}
+```
+
+Required fields to parse the response:
 
-  Required fields to parse the response:
-  - **Path for Date of Birth**: In what path of the response is the user's Date of Birth?.
+- **Path for Date of Birth**: Path in the response structure containing the user's Date of Birth?
```

Comment:
In all fields in this section, I think the question mark is no longer necessary :thinking:.

```suggestion
- **Path for Date of Birth**: Path in the response structure containing the user's Date of Birth
```